apps/workers/Dockerfile

```diff
@@ -6,8 +6,8 @@
 FROM node:16.18.1 as builder
 
 WORKDIR /app
-COPY ./dist/apps/workers ./yarn.lock ./prisma ./
-RUN yarn install --production
+COPY ./dist/apps/workers ./pnpm-lock.yaml ./prisma ./
+RUN pnpm install --production
```

Comment:
`--production` is not accepted. It has to be `--prod` or `-P`. `--frozen-lockfile` is also highly recommended